src/pages/Tap.vue

```diff
@@ -225,6 +231,7 @@ export default {
 
             if (char.match(rxNumber)) {
                 event.preventDefault();
+                1;
```

Comment:
is this `1` supposed to be here by itself?

Reply:
er. no. 😬